powersoftau/benches/accumulator.rs

```diff
@@ -4,46 +4,24 @@ use rand::thread_rng;
 use snark_utils::*;
 use zexe_algebra::Bls12_377;
 
-use powersoftau_v1::{
-    batched_accumulator::BatchedAccumulator,
-    keypair::{PrivateKey as PrivateKeyV1, PublicKey as PublicKeyV1},
-    parameters::{
-        CeremonyParams as CeremonyParamsV1, CheckForCorrectness as CheckForCorrectnessV1,
-    },
-};
-
 use test_helpers::{generate_input, setup_verify};
 
 // Benchmark comparing the generation of the iterator in parallel chunks
 // Parallel generation is strictly better
 fn generate_initial_benchmark(c: &mut Criterion) {
-    let batch = 256;
-    let mut group = c.benchmark_group(format!("generate_initial_{}", batch));
+    let mut group = c.benchmark_group("generate_initial");
     for compression in &[UseCompression::Yes, UseCompression::No] {
-        for power in 2..14 {
+        for power in 10..14 {
```

Comment:
Making only large benchmark checks with large batches, since we can afford to have much larger batches in memory